### DISCUSSION TITLE
Microsoft EdgeWebView2 Runtime required?

### DISCUSSION THREAD

**Discussion by Someonemustnothavethis** [TumblThreeApp/TumblThree#403] [Q&A (Question & Answer)]
When I fired T3 up this morning for the once-a-month download of all my blogs, it asked me to download the titled software.  I let it, and found it had downloaded both MicrosoftEdgeWebview2Setup.exe and  MicroSoft PC Manager Setup.exe
Being hysterically leery of anything from MS, especially with names that tell me it wants to 'manage' MY computer, and install that evil MS Edge stuff, I didn't run either of them.
T3 worked like a charm despite not having these 'essential' programs installed.
What are these evil and insidious programs?  
As github is now owned by MS, is this just a sleazy way to get their crapware installed on my computer?
I'm on Win7 64 bit with no intention of upgrading this particular computer.
Thanks!
Bob

**Comment by thomas694** [comment]
It's good that you don't just run anything and think for yourself.

The [link](https://go.microsoft.com/fwlink/p/?LinkId=2124703) that TumblThree opens leads to the download of "MicrosoftEdgeWebview2Setup.exe". If you run the setup the Webview2 Runtime package is downloaded and installed.
No other program is downloaded or installed. I checked it on a Win7 system here.

Are you sure the "MicroSoft PC Manager Setup" (if not mistyped by you, it contains a misspelling that makes it suspicious) was downloaded at the same time as the "MicrosoftEdgeWebview2Setup"? Please check the "Date modified" in Windows Explorer.
You can also move these two files out of your downloads folder and repeat the download again.
And you can `right-click -> Properties -> Digital Signatures` on the exe and verify (if necessary mark the row and click "Details") the name of the signer (normally "Microsoft Corporation") and that it says something like "This digital signature is OK".

I highly doubt that the other exe was downloaded at the same time and the verification passes. I guess the file was downloaded before by your internet browsing activity.

**Reply by Someonemustnothavethis** [reply]
I just tried it again and this time downloaded to a new blank directory.  All I got the MS Webview2 thing-a-ma-jig.  I haven't installed it, and won't until you tell me that it's necessary for T3 (TumbleThree, not Terminator 3) .
And you're right, I'm hysterically cautious with software as I'm a serious (*Shhh...p-r-te*).  I run new stuff in Sandboxie and I also have an old standalone laptop running Ubuntu Linux and every antivirus known to man.  
Thanks for the reply and help.
...and T3 continues to run perfectly for me.
Bob

**Reply by thomas694** [reply]
TumblThree needs the Webview2 runtime for the authenticate functionality. If you only download public blogs (that don't need the SVC crawler), it might work without, else you gonna need it.
